main.tf

```diff
@@ -23,6 +23,7 @@
 
 data "aws_route53_zone" "main" {
   name = "${var.zone_name}"
+  private_zone = "${var.private_zone}"
```

Comment:
How is ACM able to validate the SSL certificate if the hosted zone is private? This module creates DNS records in the hosted zone that ACM can then validate to confirm the domain is in fact yours. Those records need to be accessible on the public internet to be validated.

Reply:
Yea, I actually ran into that very issue --though I was able to manually approve the certificate in the console (it was shown as pending approval), and terraform picks up on that approval and uses that for the validation resource in its state. We have a particular need at the moment to keep an application internal to the organization, thus our need for the private hosted zone. I copied the module locally and added the flag, still have some thinking to do about this. Thanks for your reply.